src/tests.builds

```diff
@@ -3,12 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <ItemGroup>
-    <TestProjectExclusions Include="$(MSBuildThisFileDirectory)*\tests\**\System.Security.AccessControl.Tests.csproj" />
     <TestProjectExclusions Include="$(MSBuildThisFileDirectory)*\tests\**\WebServer.csproj" Condition="'$(OS)' != 'Windows_NT'" />
```

Comment:
I'm fine if you do it with this PR or a separate one but I would like you as opposed to @chcosta to take a look at actually fixing those projects.